students/MikeShand/lesson04/dict_lab.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+dictionary = {"name": "Chris", "city": "Seattle", "cake": "chocolate"}
+
+print(dictionary)
+
+dictionary.popitem()

```

Comment:
I would probably get used to using `pop()` vs `popitem()`, 

`pop` can do everything `popitem` can, plus more